operations/flags.go

```diff
@@ -206,6 +207,13 @@ func addDbSettingsFlags(flags ...cli.Flag) []cli.Flag {
 	)
 }
 
+func addCommittedOnlyFlag(flags ...cli.Flag) []cli.Flag {
+	return append(flags, cli.BoolFlag{
+		Name:  joinFlagNames(committedFlagName, "c"),
```

Comment:
Will this conflict with the top-level `--conf` flag's short form of `-c`?

Reply:
I think it will only conflict if it's added to the same command (the conf flag is just for service/web, service/deploy/transform, and service/deploy/anser)

Reply:
On second thought: I removed the -c short form because it will conflict.